### PR TITLE
Show groupfolders in shared folders

### LINKED ISSUE #129: Support Groupfolders
We use group folders app for shared space and most apps support them without any issue, but new photo app won't show any folders/images.

### PR DESCRIPTION
**Pull request by rullzer**
Fixes #129 

So. The code is simple. But I did notice it taking longer on my system. Also mainly because group folders then to be freaking huge ;)

CC: @xf- @gabrielklein @jakobroehrl 

But should be fine in the 99% case.

### CODE REVIEW

File: lib/Controller/AlbumsController.php [hunk 1]
```diff
@@ -136,7 +135,8 @@ private function validFile(File $file, bool $shared): bool {
 	}
 
 	private function isShared(Node $node): bool {
-		return $node->getStorage()->instanceOfStorage(SharedStorage::class);
+		return $node->getStorage()->instanceOfStorage(SharedStorage::class) ||
+			$node->getStorage()->instanceOfStorage(\OCA\GroupFolders\Mount\GroupFolderStorage::class);
```
Comment (skjnldsv) [inline lib/Controller/AlbumsController.php:139]: `use OCA\GroupFolders\Mount\GroupFolderStorage` ?

Comment (rullzer) [reply lib/Controller/AlbumsController.php:139]: No lets not import it with use. As we can't know for sure it is available.
This just expands to the classname.

**Pull request merged**